package/MDAnalysis/analysis/pca.py

```diff
@@ -355,12 +356,12 @@ def n_components(self, n):
                 n = len(self._variance)
             self.results.variance = self._variance[:n]
             self.results.cumulated_variance = (np.cumsum(self._variance) /
-                                       np.sum(self._variance))[:n]
+                                               np.sum(self._variance))[:n]
             self.results.p_components = self._p_components[:, :n]
         self._n_components = n
 
     def transform(self, atomgroup, n_components=None, start=None, stop=None,
-                  step=None):
+                  step=None, verbose=None):
```

Comment:
Set the default to `False` because it's not clear what `None` will do from the docs. If it behaves like `False` then be clear and set it to `False` in the first place.